src/datagen/main/java/mekanism/common/recipe/impl/OreProcessingRecipeProvider.java

```diff
@@ -64,6 +64,11 @@ public void addRecipes(RecipeOutput consumer, HolderLookup.Provider registries)
               IngredientCreatorAccess.infusionStack().from(MekanismAPITags.InfuseTypes.CARBON, 10),
               MekanismItems.ENRICHED_IRON.getItemStack()
         ).build(consumer, Mekanism.rl(basePath + "iron/enriched"));
+        ItemStackChemicalToItemStackRecipeBuilder.metallurgicInfusing(
+              IngredientCreatorAccess.item().from(MekanismTags.Items.PROCESSED_RESOURCES.get(ResourceType.DUST, PrimaryResource.IRON)),
```

Comment:
Do we want to add a second recipe like this, or just make the first recipe use an an or (compound) ingredient? I am fine with either. If you do decide to use an or ingredient you can use one of our helpers: `BaseRecipeProvider.createIngredient`, which you can see how we do it in how we add the recipes that create carbon.

I sort of think it makes more sense for it to just be a single recipe. But I am leaving that up to you.

Reply:
Maybe we can merge it later if we remember. Separate for now would promote visibility?

Reply:
Up to you, though I don't think the visibility is that bad given the fact there is basically only ever one iron ingot in modded playthroughs, and then it just cycles between the two inputs.

Reply:
I think I am going to merge them for now. Visibility doesn't seem that bad to me for the above stated reason

Reply:
Actually scratch that, I didn't think about the fact that whil ethere is only one iron ingot, there are often multiple iron dusts.